src/Corrade/Utility/Path.cpp

```diff
@@ -640,11 +647,18 @@ Containers::Optional<Containers::String> configurationDirectory(const Containers
 
     /* Windows (not Store/Phone) */
     #elif defined(CORRADE_TARGET_WINDOWS) && !defined(CORRADE_TARGET_WINDOWS_RT)
-    /** @todo get rid of MAX_PATH */
-    wchar_t path[MAX_PATH];
+    wchar_t* path = nullptr;
+    Containers::ScopeGuard guard{path,
+        #ifdef CORRADE_MSVC2015_COMPATIBILITY
+        /* MSVC 2015 is unable to cast the parameter for CoTaskMemFree */
+        [](LPVOID path){ CoTaskMemFree(path); }
```

Comment:
I think 

```suggestion
        [](wchar_t* path){ CoTaskMemFree(path); }
```

could fix the MSVC2015 failures? Same in the other case.

Reply:
Oh, sorry I didn't notice it earlier. I'll fix that. 😅